pynitrokey/fido2/dfu.py

```diff
@@ -45,30 +51,30 @@ def find_all():
 
 
 class DFUDevice:
-    def __init__(
-        self,
-    ):
+    def __init__(self):
         pass
 
     @staticmethod
-    def addr2list(a):
+    def addr2list(a: int) -> List[int, int, int, int]:
```

Comment:
`List` only has one type parameter.  If you want to add the length to the type annotation, you’d have to use a tuple.

Reply:
mmmh immutable `List` -> nope!
so, changing to `List[int]` as the function actually returns a list (even though it might semantically be a `Tuple`